_posts/randy/2022-07-24-how-video-takes-all-day.md

```diff
@@ -0,0 +1,189 @@
+
+There is no arbitrary clip length limit on the internet. Streaming services like YouTube can easily neak in short ad snacks; you are forced to watch while fumbling for the ad kill button. 
+
+Video writers and producers of late need to pack whole shows; truncated messages into shorter time frames. Making things smaller is usually harder, but edit sweets will abound to fill demands of the moment - however brief. 
+
+**TIME FILLS SPACE KILLS TIME** 
+
+In video production circles, all runtimes - seconds and minutes are not created equal. One can get lucky capturing a happenstance clip of a [monkey smelling butt-finger](https://www.youtube.com/watch?v=t4A1tWzlJW8{:target="_blank"}), or lose a few years hiding in the jungle trying to film the sequel. Strike the jungle shoot before it starts to sound like a good idea. 
+
+The first consideration in any project should be to **determine an achievable goal within the timeframe assigned.** Analog Sampling's video format outline dictates what goals are going to be practical within our modest project case: 
+
+     maximum runtime 10 seconds (for no good reason) 
+
+          no budget; no travel; no help; no complaints 
+          create concepts and elements as needed 
+          use existing media library and free online content 
+          consistent start-end frames for semi-seamless looping 
+
+     maximum completion time 1 day 
+
+With these narrow guidelines, ideal but budget-busting production teams will be forever jettisoned in favor of a one-head show. Who deserves a crew for a home-made 10 second piece? There will not be a middle-person involved to moderate this soup to nuts role. No one to share duties with but no one else to argue with (other split-personality or consciences exempted), but then no one else to blame. 
+
+With the broadcast air of expectation slot to fill, it boils down to a matter of time to make something that matters. 
+
+**CREATIVE CHICKEN – WHAT COMES FIRST?** 
+
+Writing a reasonably cohesive technology article is one skill to aspire to, but creating a short representative video clip to represent meandering words is another challenge altogether. 
+
+Some subjects are hard to visualize – other creative treatments can hatch out of nowhere. A single picture can say a thousand words. Here we have 300 picture frames to work with - upward potential for 300,000 words communicated inside a 10 second video. 
+
+Which of these words stand out to tell the tale in short? It helps to be the writer and have familiarity with the subject matter, but there Creative Gods can be nudged along with a few words of recollection. Thy ask: 
```

Comment:
there seems out of place here.  At first I thought it should be "their", but that also seems weird, so I think I am not comprehending what this sentence is trying to say.

Reply:
the

Reply:
focus